src/Bridge/Doctrine/Orm/Filter/ExistsFilter.php

```diff
@@ -30,11 +34,24 @@
  * Request: GET /products?brand[exists]
  * Interpretation: filter products which have a brand
  *
- * @author Teoh Han Hui <teohhanhui@gmail.com>
+ * @author Julien Verger <julien.verger@gmail.com>
```

Comment:
Please add you after the previous author.

Reply:
done.